lib/build.ts

```diff
@@ -258,19 +261,21 @@ async function checkAllowDockerPlatformHandling(
 			// eat the exception... yummy!
 		}
 
-		console.log(
-			`Image manifest data unavailable for ${repository}. No support for specifying platform.`,
+		debug(
+			`${task.serviceName}: Image manifest data unavailable for ${repository}`,
 		);
 		// at this time, treat any errors as false (no platform support)
 		return false;
 	};
 
 	if (task.imageName) {
-		imageReferences = [task.imageName + !!task.tag ? `:${task.tag}` : ''];
+		imageReferences.push(task.imageName);
```

Comment:
This is a bug fix:
* `task.imageName` is the name of an "external image" when the `docker-compose.yml` file contains, for example, an `image: debian:10` line for a service, instead of a `build:` line/section. The value of `task.imageName` is something like `debian:10`, including the tag part.
* `task.tag` is used to tag the local images that are built (with a Dockerfile) on the machine where Docker is running. For a project name 'myFleet' and a service name 'myService', the value of `task.tag` would be `myfleet_myservice`.

So, in this example, putting together `task.imageName` and `task.tag` would result in `debian:10:myfleet_myservice`, which isn't right. Except that there was also a bug within the bug, in that:
```
task.imageName + !!task.tag ? `:${task.tag}` : ''
```
associates as:
```
(task.imageName + !!task.tag) ? `:${task.tag}` : ''
```
rather than the semantically intended:
```
task.imageName + (!!task.tag ? `:${task.tag}` : '')
```

Reply:
thank you so much for catching this!